test/dotnet-core-uninstall.Tests/Shared/VSVersioning/VSVersionTests.cs

```diff
@@ -27,8 +27,8 @@ public class VSVersionTests
         [InlineData(new string[] { "2.2.100", "2.2.200" }, new bool[] { false, false })]
         [InlineData(new string[] { "2.2.100", "2.2.200", "2.2.300" }, new bool[] { false, true, false })]
         [InlineData(new string[] { "3.0.0", "3.0.1", "5.0.0" }, new bool[] { true, true, false })]
-        [InlineData(new string[] { "5.0.0", "5.0.1", "6.0.1" }, new bool[] { true, false, true })]
-        [InlineData(new string[] { "6.0.0", "6.0.1", "7.0.0" }, new bool[] { true, true, false })]
+        [InlineData(new string[] { "6.0.0", "6.0.1", "7.0.1" }, new bool[] { true, false, false })]
```

Comment:
@baronfel or @joeloff,

This is the part I was least confident in as far as test changes. From what I can tell, if you provide a certain set of versions, it _always_ says the highest version is required. So how could the "allowed" list ever end in true? I made it false.

Reply:
I did note that main built, so I think there's something I'm missing here. Just wanted to call out my confusion.

Reply:
We need to dig into this as this doesn't match the pattern i would expect from past history. I'd have to run the test to know for sure why it works this way.